README.md

```diff
@@ -1,6 +1,10 @@
 # cost-analyzer helm chart
 Helm chart for the Kubecost project, which is created to monitor and manage Kubernetes resource spend. Please contact team@kubecost.com or visit [kubecost.com](http://kubecost.com) for more info.
 
+While Helm is the recommended install path, these resources can also be created with the following command: 
```

Comment:
Add something like "the values used in the kubecost.yaml manifest are from the defaults in values.yaml"